CODEOWNERS

```diff
@@ -463,6 +463,7 @@
 /bundles/org.openhab.voice.voskstt/ @GiviMAD
 /bundles/org.openhab.voice.watsonstt/ @GiviMAD
 /bundles/org.openhab.voice.whisperstt/ @GiviMAD
+/bundles/org.openhab.voice.openaitts/ @Artur-Fedjukevits
```

Comment:
Please move this up before `org.openhab.voice.picotts` so that the list is in alphabetic order.